plugins/hubspot/src/api.ts

```diff
@@ -80,6 +223,22 @@ interface RequestOptions {
     body?: any
 }
 
+const PROXY_URL = "https://framer-cors-proxy.framer-team.workers.dev/?"
+const API_URL = "https://api.hubapi.com"
+
+export const queryKeys = {
+    blogPosts: (limit: number, properties: string[]) => ["blog-posts", limit, properties] as const,
+    publishedTables: (limit: number) => ["hubdb-tables", limit] as const,
+    publishedTable: (tableId: string) => ["hubdb-table", tableId] as const,
+    tableRows: (tableId: string, properties: string[], limit: number) =>
+        ["hubdb-table-rows", tableId, properties, limit] as const,
+    user: () => ["user"] as const,
+    account: () => ["account"] as const,
+    inboxes: () => ["inboxes"] as const,
+    forms: () => ["forms"] as const,
+    meetings: () => ["meetings"] as const,
+} as const
```

Comment:
I've made them inline and turned off caching for the queries using `cachedFetch` using `staleTime: 0`.  

But From what I understand, `useQuery` and `cachedFetch` both have the same cache since they use the same `queryClient`?